### DISCUSSION TITLE
Xteve update docker

### DISCUSSION THREAD

**Discussion by deku-m** [alexbelgium/hassio-addons#347] [General]
Hi Alex,

I was looking into xteve and i saw someone who did some intense updates on the application,
Was wondering if you could create a beta from this 
https://github.com/SCP002/xTeVe

And if it is ok replace the old one as that one isnt updated for months after testing?

Thanks,

**Comment by alexbelgium** [comment]
Hi, this seems nice but this repo is archived, which surely means that there won't be any other updates. Still interested? If yes I'll update the addon from my repo in a test version so you can try if it works

**Reply by deku-m** [reply]
Yes the repo is archived indeed but he made a lot of changes/updates. Still looking if someone is continuing this but he is the only one at this moment. Can you make a test version so i can try?

**Comment by alexbelgium** [comment]
Test version pushed with just the upstream image changed, let's see how that goes 
Edit : it doesn't build, it won't be a simple plug and play

**Comment by alexbelgium** [comment]
Ah, I understand why it fails : it only exists for amd64. Is that the arch you use?

**Comment by alexbelgium** [comment]
Would this version be acceptable? It was last updated as recently as the one you propose, but is multi arch : tinyserve/xteve

**Reply by deku-m** [reply]
i dont know about the version he is running but ok by me.

**Comment by deku-m** [comment]
I tested the 2.5 version but seems a bug is in the application itself from scp too bad i had high hopes.
The build was ok btw.

Can you revert to the old one?

**Comment by alexbelgium** [comment]
I'll test another new version. Could you please test it? If it doesn't word I'll fully revert. Thanks

**Reply by deku-m** [reply]
I tested it this version is working but its not the new XTEVE version from SCP or so its 2.2.0.
Just the original version. But the docker env looks maintained maybe thats important.

The issue with the SCP version is related to the xteve application i think or he forgot something in the docker.
As it is showing after filling in the m3u link and EPG he neeeds to create a database for it but hangs on setup.
So it stays at updating database. Saw someone posting the same issue as well.

**Comment by alexbelgium** [comment]
Ah well... Then for the moment I'll leave this new image that seems much more recent than the previous one

**Reply by deku-m** [reply]
Can you place the SCP VERSION still online so i can check if it really is a small issue im curious as someone said it works after couple of times is it possible to create it as a second xteve add-on?

**Reply by alexbelgium** [reply]
Hi, I've put the SCP version of the addon here : https://github.com/alexbelgium/hassio-addons-test

If you find a way to make it work, please propose a PR to make the change directly in the addon! Then I would push it to the main repo. Otherwise for the moment I'll simply keep it as it is

Have fun!

**Reply by deku-m** [reply]
Hi Alex,

Could you make a new test docker and change the source to https://github.com/SenexCrenshaw/xTeVe  ?
Thanks in advance

**Comment by alexbelgium** [comment]
Hi, sorry he doesn't seem to provide images all docker links refer to the original repo

**Reply by deku-m** [reply]
There are no image indeed but only the docker file to build the hub link isn't available that was written in docker file.
But this can't be build then by using docker file?

**Comment by alexbelgium** [comment]
It's much harder and I avoid building addons that way. I'll look at it when I have some time
